image_rotate/package.xml

```diff
@@ -28,30 +28,27 @@
   <license>BSD</license>
   <url>http://ros.org/wiki/image_rotate</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <test_depend>ament_cmake_gtest</test_depend>
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_lint_common</test_depend>
 
-  <test_depend>rostest</test_depend>
-  
-  <build_depend>cmake_modules</build_depend>
   <build_depend>cv_bridge</build_depend>
-  <build_depend>dynamic_reconfigure</build_depend>
-  <build_depend>geometry_msgs</build_depend>
+  <build_depend>class_loader</build_depend>
   <build_depend>image_transport</build_depend>
-  <build_depend>nodelet</build_depend>
-  <build_depend>roscpp</build_depend>
+  <build_depend>rclcpp</build_depend>
   <build_depend>tf2</build_depend>
   <build_depend>tf2_geometry_msgs</build_depend>
   <build_depend>tf2_ros</build_depend>
```

Comment:
`<build_depend>` and `<exec_depend>` can be combined with `<depend>` in Package Format 2 (and 3).

Reply:
fixed